src/Apps/Collect2/Routes/Collection/Components/CollectionsHubRails/FeaturedCollectionsRails/index.tsx

```diff
@@ -49,12 +49,12 @@ export const FeaturedCollectionsRails: React.FC<Props> = ({
     })
   }
 
-  const renderCarousel = ({ xs, sm, xl }) => {
+  const renderCarousel = breakpoints => {
```

Comment:
Rather than passing an `{}` down below, `breakpoints` can have a default argument

Reply:
That would be something like `renderCarousel = (breakpoints = {}) => ...` right? 

I tried that and was getting a type error: `Property 'xs' does not exist on type '{}'.`

Reply:
Yup, that's correct; might want to play with the type a bit as it's looking for an interface 👍